### ISSUE TITLE
Extract styles to css

### ISSUE THREAD

**Issue by unitehenry** [striven-erp/striven-editor#30]
Take any ```el.style.[style]``` and extract it to a css class.

```js
el.classList.add('appropriate-class');
```

**Comment by Brostafa** [comment]
The description is a bit vague. Do you mean something like:

```html
<div style="font-size: 30px; display: inline-block;" id="foo"></div>
```

```javascript
const selector = '#foo'
const styles = extractCss(selector)
// styles {
// 	'font-size': '30px',
// 	display: 'inline-block'
// }

// we grab the output of extractCss and add it as input for addCssClass
addCssClass(selector, styles)
```

- Result html

```html
<div style="font-size: 30px; display: inline-block;" id="foo" class="font-size-30px display-inline-block"></div>
```

**Comment by unitehenry** [comment]
Sorry @Brostafa , I should've been clearer. I didn't mean pragmatically extract the styles.

Something more along the lines of this...

In the ```striveneditor.js``` source code, you will see all the ```DOM Node```s look like this.

```js
const aNewNode = document.createElement('div');
aNewNode.style.fontSize = '30px';
``` 

We want to use class-based css styling like this.

```js
const aNewNode = document.createElement('div');
aNewNode.classList.add('node-style');
```

```css
.node-style {
     font-size: 30px;
}
```

We can import these styles like so.

```js
import './striveneditor.css';
```

**Comment by Brostafa** [comment]
Gotcha, please assign to me.

**Comment by Brostafa** [comment]
Done https://github.com/striven-erp/striven-editor/pull/32

While working on this I have encountered two things: 

1- symlink doesn't work https://github.com/striven-erp/striven-editor/blob/master/CONTRIBUTING.md#symlink-development-package `npm link striven-editor` that didn't work for me. However, this did `npm link @striven-erp/striven-editor`

2- The version you have of editor on the master branch don't have toolbar animation working (not sure if it was intended). For instance if you went to https://striven-erp.github.io/striven-editor/ and click on editor -> toolbar will be animated at the bottom but master branch doesn't have it.

**Comment by unitehenry** [comment]
@Brostafa Thank you so much for pointing these issues out!

I've opened an issue to update ```CONTRIBUTING``` here: #37 

I'm aware that the branches are a l little out of whack. Currently, the ```master``` branch is past the ```npm publish``` version, and the dev branch is not being utilized. I've already addressed this in the ```CONTRIBUTING``` doc.

From now on, we will be developing on the ```test``` branch and making PRs to the ```dev``` branch. I will merge to master on time of publish.

Thanks for your help 😄
